Detector/DetFCChhBaseline1/compact/FCChh_DectDimensions.xml

```diff
@@ -65,58 +65,55 @@
     <constant name="FwdTracker_z2" value="2000*mm"/>
     <constant name="FwdTracker_zOffset" value="14000*mm"/>
```

Comment:
Hi! @drasal, seems like we are currently using a `zOffset` of 10000 mm and a `z2` of 6000 mm. I don't know which one is the most recent config!

![](http://fcc-tklayout.web.cern.ch/fcc-tklayout/FCChh_Option3.v02/img053.png)